l10n_it_fatturapa_in/tests/data/IT02780790107_11004.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- pylint:disable=xml-syntax-error -->
-<p:FatturaElettronica versione="FPA12" xmlns:ds="http://www.w3.org/2000/09/xmldsig#&quot;"
```

Comment:
Attenzione che il file `IT02780790107_11004.xml` è valido per SDI, quindi potrebbe arrivare e quindi deve poter essere importato in odoo

Reply:
Ne abbiamo parlato, il validatore lo rifiuta perché non è un URI valido. Per altro non è quello l'URN corretto del namespace, non è che puoi mettere quello che vuoi. Diciamo che lo SdI se lo passa non lo verifica.
 Va gestito a mano con una re, prima del parsing.

Reply:
Ok, se quell'XML da SDI può arrivare, deve essere testato e importabile